sycl/source/detail/scheduler/commands.cpp

```diff
@@ -499,8 +499,8 @@ void Command::processDepEvent(EventImplPtr DepEvent, const DepDesc &Dep) {
   // TODO temporarily disabled with Level Zero since the enqueued operations
   // that are implemented directly in the plugin (e.g. map/unmap) do not satisfy
   // in-order queue requirements.
-  if (WorkerQueue->is_host() ||
-      WorkerQueue->getPlugin().getBackend() != backend::level_zero)
+  /*if (WorkerQueue->is_host() ||
```

Comment:
please remove the commented check and updat ethe comment at line 499

Reply:
yes, this is uploaded by mistake. removing it.

Reply:
do you want me to fix this in this PR? It was added by https://github.com/intel/llvm/pull/3188

Reply:
done.